craps/table/bets.py

```diff
@@ -0,0 +1,480 @@
+
+    def get_signature(self):
+        return BetSignature(type=self.__class__, wager=self.wager, odds=self.odds,
+                            placement=self.placement)
+
+
+@ignore_placement_for_compare
+class Horn(PropBetAbstract):
+    """Multi-Bet: Horn Numbers"""
+    multi_bet = 4
+
+    def is_winner(self, outcome: DiceOutcome):
+        return outcome.total() in [2, 3, 11, 12]
+
+    def get_payout(self, outcome: DiceOutcome) -> int:
+        if not self.is_winner(outcome):
+            return 0
+        return int(self.wager / self.multi_bet * (
+            self._table.config.hop_hard_pay_to_one if outcome.is_hard()
+            else self._table.config.hop_easy_pay_to_one))
+
+
+class HornHigh(Horn):
+    """Multi-Bet: Horn Numbers with 1/5 placed on a specific number"""
+    multi_bet = 5
+
+    def _check_valid(self):
+        if not self.placement:
+            raise InvalidBetException(f'{self.__class__.__name__} bet requires a location')
+        if self.placement not in [2, 3, 11, 12]:
+            raise InvalidBetException(
+                f'{self.placement} is not a valid location for a {self.__class__.__name__} bet')
+
+    def get_payout(self, outcome: DiceOutcome) -> int:
+        if not self.is_winner(outcome):
+            return 0
+        if outcome.total() == self.placement:
+            return int(self.wager / self.multi_bet * 2 * (
+                self._table.config.hop_hard_pay_to_one if outcome.is_hard()
+                else self._table.config.hop_easy_pay_to_one))
+        return int(self.wager / self.multi_bet * (
+            self._table.config.hop_hard_pay_to_one if outcome.is_hard()
+            else self._table.config.hop_easy_pay_to_one))
+
+
+@ignore_placement_for_compare
+class World(PropBetAbstract):
+    """World Bet: Horn Bet or AnySeven Bet"""
+    multi_bet = 5
+
+    def is_winner(self, outcome: DiceOutcome):
+        return outcome.total() in [2, 3, 11, 12]
+
+    def is_loser(self, outcome: DiceOutcome):
+        return outcome.total() not in [2, 3, 7, 11, 12]
+
+    def get_payout(self, outcome: DiceOutcome) -> int:
+        if not self.is_winner(outcome):
+            return 0
+        return int(self.wager / self.multi_bet * (
+            self._table.config.hop_hard_pay_to_one if outcome.is_hard()
+            else self._table.config.hop_easy_pay_to_one))
+
+
+@ignore_placement_for_compare
+class Craps3Way(PropBetAbstract):
+    """Multi-Bet: 3-Way Craps (Hop the craps numbers)"""
+    multi_bet = 3
+
+    def is_winner(self, outcome: DiceOutcome):
+        return outcome.total() in [2, 3, 12]
+
+    def get_payout(self, outcome: DiceOutcome) -> int:
+        if not self.is_winner(outcome):
+            return 0
+        return int(self.wager / self.multi_bet * (
+            self._table.config.hop_hard_pay_to_one if outcome.is_hard()
+            else self._table.config.hop_easy_pay_to_one))
+
+
+@ignore_placement_for_compare
+class CE(PropBetAbstract):
```

Comment:
CE is not a very descriptive name, what about
```suggestion
class CrapsOrEleven(PropBetAbstract):
```

Reply:
The actual bet is called the C&E bet

Reply:
In that case, it's just a terrible name all together but it at least makes more sense now.